app/views/candidates/schools/_phase2.html.erb

```diff
@@ -13,7 +13,7 @@
 
       <dl class="govuk-summary-list govuk-summary-list--no-border">
         <%= dlist_item 'Individual requirements:', id: 'individual-requirements' do %>
-          <%= content_or_msg @presenter.individual_requirements, "No requirements specified" %>
+          <%= content_or_msg @presenter.individual_requirements, "No requirements specified", display_as_list: true %>
```

Comment:
I think this is combining too much logic - `content_or_msg` needs to behave differently here

how about dropping the changes to `summary_row` and to `content_or_msg` and just replacing this line with

```erb
<%= content_or_msg split_to_list(@presenter.individual_requirements), "No requirements specified" %>
```